### PR TITLE
Assistant welcome ux static

### PR DESCRIPTION
**Pull request by PopDaph**
Layout for New conversation page. 
Buttons are not working and for now we get 4 random agents, this PR is only about the layout. 

Next step: 
- configure the global @helper agents and make the button to start a convo with it work.
- display the expected list of features agents.
- build a page to display the list of agents. 

Capture d’écran 2023-09-20 à 12 02 40

### CODE REVIEW

File: front/pages/w/[wId]/assistant/new.tsx [hunk 1]
```diff
@@ -148,34 +149,86 @@ export default function AssistantNew({
...
+                </span>
+              </div>
+              <p className="text-element-700">
+                Dust is not just a single assistant, it’s a full team at your
+                service.
+                <br />
+                Each member has a set of specific set skills.
+              </p>
+              <p className="text-element-700">
+                Meet some of your Assistants team:
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:189]: Can we try adding a button see all here?

**Review by spolu — CHANGES_REQUESTED** [review]
LGTM can we remove the back-end logic and useSwr + client-side logic for featuring

File: front/pages/w/[wId]/assistant/new.tsx (hunk 1 above)
Comment (PopDaph) [reply front/pages/w/[wId]/assistant/new.tsx:193]: Done!

File: front/lib/api/assistant/configuration.ts [hunk 2]
```diff
@@ -188,6 +188,22 @@ export async function getAgentConfigurations(
   return [...globalAgents, ...agents];
 }
 
+export async function getFeaturedAgentConfigurations(
```
[the remark this answers is not in the document]
Comment (PopDaph) [reply front/lib/api/assistant/configuration.ts]: Done!

**Comment by PopDaph** [comment]
### 4 assistants: 
Capture d’écran 2023-09-20 à 14 06 42

### All assistants: 
Capture d’écran 2023-09-20 à 14 06 51

**Review by spolu — APPROVED** [review]

**Comment by Duncid** [comment]
I think it's better to open a gallery of assistants in a modal than in the context!

**Pull request merged**